### ISSUE TITLE
[BUG] Emotes showing untextured cubes

### ISSUE THREAD

**Issue by yorik100** [PluginBugs/Issues-ItemsAdder#2708]
### Terms

- [X] I'm using the very latest version of ItemsAdder and its dependencies.
- [X] I am sure this is a bug and it is not caused by a misconfiguration or by another plugin.
- [X] I already searched on this [Github page](https://github.com/PluginBugs/Issues-ItemsAdder/issues) to check if the same issue was already reported.
- [X] I already searched on the [plugin wiki](https://itemsadder.devs.beer/) to know if a solution is already known.
- [X] I already searched on the [forums](https://forum.devs.beer/) to check if anyone already has a solution for this.

### Discord tag (optional)

yorik100#5011

### What happened?

Emotes show untextured blocks

### Steps to reproduce the issue

1. Get latest Spigot
2. Get latest ItemsAdder
3. Get latest LoneLib
4. Get latest ProtocolLib
5. Connect to the server
6. Do /iazip
7. Do /iareload
8. Texture is now invalid for emotes

### Server version

CraftBukkit version 3762-Spigot-7d7b241-2270366 (MC: 1.19.4) (Implementing API version 1.19.4-R0.1-SNAPSHOT)

### ItemsAdder Version

3.4.1f

### ProtocolLib Version

5.1.0-SNAPSHOT-645

### LoneLibs Version

1.0.24

### FULL server log

[latest.log](https://github.com/PluginBugs/Issues-ItemsAdder/files/11655266/latest.log)

### Screenshots/Videos (you can drag and drop files or paste links)

https://streamable.com/fuqixu

Note 1 : It always show the update available message despite me being on latest version
Note 2 : Yesterday, I did /iazip on the version prior to this one and it didn't show missing texture blocks, it showed steve skin despite me being on 100% vanilla and only having ItemsAdder and its 2 dependencies, missing texture started occuring on the first /iazip I did on latest version, I did not try to /iazip on previous version to see if the problem comes from the newest version

**Comment by yorik100** [comment]
> Downgrade itemsadder, run /iazip and check if it works or not. Anyway please send here your CLIENT log of when the issue occurred, here how to get it: https://itemsadder.devs.beer/faq/identify-why-textures-are-not-shown

It magically worked, fixed both the no texture and the steve texture issue, only difference is that I switched from Spigot to Paper, switching from Spigot to Paper seems to fix the Steve skin issue, at least it did for me

**Comment by LoneDev6** [comment]
If you're using an offline mode server you have to install SkinsRestorer.
It was tested and working fine with SkinsRestorer.

I also tested on online mode server and it shows the correct skin.
So the issue might rely somewhere else.

**Comment by yorik100** [comment]
> If you're using an offline mode server you have to install SkinsRestorer. It was tested and working fine with SkinsRestorer.
> 
> I also tested on online mode server and it shows the correct skin. So the issue might rely somewhere else.

I downloaded latest Spigot and ran in online mode with LoneLib and ProtocolLib
The Steve skin issue is fixed for sure on my side by switching from Spigot to PaperSpigot
The untextured issue probably comes from the newest update

**Comment by LoneDev6** [comment]
Please be more specific.
Is the black and purple issue reproducible in the new update only?
This is totally not clear to me.

In order to test that you have to downgrade to the previous itemsadder version, then run /iazip, then check if the issue is still persisting.
Then update to the latest version, run /iazip again, check if the issue persists.

Thanks.

**Comment by yorik100** [comment]
> Please be more specific. Is the black and purple issue reproducible in the new update only? This is totally not clear to me.
> 
> In order to test that you have to downgrade to the previous itemsadder version, then run /iazip, then check if the issue is still persisting. Then update to the latest version, run /iazip again, check if the issue persists.
> 
> Thanks.

I just said that it worked fine by downgrading itemsadder version

**Comment by yorik100** [comment]
after using iazip and iareload with 3.4.1f (latest)
![image](https://github.com/PluginBugs/Issues-ItemsAdder/assets/16320315/ff321ca6-d648-445b-a654-81305749dc9f)
after using iazip and iareload with 3.4.1e (previous version, 1 version below latest)
![image](https://github.com/PluginBugs/Issues-ItemsAdder/assets/16320315/0d0a4572-3e29-453e-b2f1-14b68d4a0e0f)

Doing /iazip with latest version glitches out emotes

**Comment by LoneDev6** [comment]
Does running `/iacleancache items` and then `/iazip` on the latest version fixes the issue or not?

**Comment by yorik100** [comment]
> Does running `/iacleancache items` and then `/iazip` on the latest version fixes the issue or not?

No, it does not, no matter what I do, as soon as I do /iazip and I reload on latest version, emotes break

**Comment by LoneDev6** [comment]
Thanks!
I'll post an hotfix for that in some minutes
